AGENTS.md

```
@@ -1 +1,114 @@
+7. Gemini reports status in this file
+8. User returns to Kiro
+
+### Gemini → Kiro Handoff
+1. Gemini commits with message: `feat(android): <description>`
+2. Gemini updates status below
+3. User returns to Kiro IDE
+4. Kiro pulls changes and continues
+
+## Status Updates
+
+### Gemini Status
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)